sql/src/main/java/io/crate/expression/symbol/format/SymbolPrinter.java

```diff
@@ -179,15 +179,33 @@ private void printGenericFunction(Function function, SymbolPrinterContext contex
             }
         }
 
-        private void printAnyOperator(Function function, SymbolPrinterContext context) {
+        private static boolean extractIgnoreCase(Function function) {
+            boolean ignoreCase = false;
+            if (function.arguments().size() == 3) {
```

Comment:
Seems like another sign to me that a dedicated `ilike` function might be cleaner.